force-app/main/default/lwc/templateTableDetails/templateTableDetails.js

```diff
@@ -0,0 +1,1131 @@
+import { LightningElement, track, api, wire } from 'lwc';
+import { loadStyle } from 'lightning/platformResourceLoader';
+import rte_tbl from '@salesforce/resourceUrl/rte_tbl';
+import dexcpqcartstylesCSS from '@salesforce/resourceUrl/dexcpqcartstyles';
+import saveDocumentTemplateSectionDetails from '@salesforce/apex/SaveDocumentTemplatesection.saveDocumentTemplateSectionDetails';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import deletetemplate from '@salesforce/apex/SaveDocumentTemplatesection.deletetemplate';
+import getAllPopupMessages from '@salesforce/apex/PopUpMessageSelector.getAllConstants';
+import gettemplatesectiondata from '@salesforce/apex/SaveDocumentTemplatesection.gettemplatesectiondata';
+import getContentVersions from '@salesforce/apex/FooterClass.getContentVersions';
+import getSearchedContentVersions from '@salesforce/apex/FooterClass.getSearchedContentVersions';
+
+export default class TemplateTableDetails extends LightningElement {
+
+    @track tableOnLoad = true;
+    @track tableDisplayed = false;
+    @track isModalOpen = false;
+    @track selectedRow = '';
+
+    isAllBorders = false;
+    isNoBorders = false;
+    isOutsideBorders = false;
+    isOutsideThickBorders = false;
+    isOutsideThickAllBorders = false;
+
+    isClearTable = false;
+    showmergefield = false;
+    showImageModal = false;
+    isHeaderMergeField = false;
+
+    imageUrls = [];
+    mainimageUrls = [];
+    @track divContentArray = [];
+    selectedimageid;
+    @track selectedImageHeight = "75px";
+    @track selectedImageWidth = "75px";
+    showText = true;
+    searchData;
+    imagebuttonlabel = 'Select Image';
+    showimages = false;
+    imageselected = false;
+    selectedimageurl;
+    imagesfound = false;
+
+    @api selectedObjectName;
+    showStatement = false;
+    showLstOfObj = false;
+
+    @track noBordersIcon = '';
+    @track allBordersIcon = '';
+    @track outsideBordersIcon = '';
+    @track thickOutsideBordersIcon = '';
+    @track thickOutsideAllBordersIcon = '';
+
+
+    fontsize = "10px";
+    rowFontSize = "10px";
+
+    fontsizeoptions = [
+        { value: '8px', label: '8' }, { value: '9px', label: '9' }, { value: '10px', label: '10' }, { value: '12px', label: '12' },
+        { value: '13px', label: '13' }, { value: '14px', label: '14' }, { value: '15px', label: '15' },
+    ];
+
+    fontfamily = "Verdana";
+    fontfamilyoptions = [
```

Comment:
All font options listed may not be supported during the PDF generation. Please refer this and update the font options.[https://developer.salesforce.com/docs/atlas.en-us.pages.meta/pages/pages_output_pdf_supported_fonts.htm](url)